internal/tofu/schemas.go

```diff
@@ -72,27 +74,38 @@ func (ss *Schemas) ProvisionerConfig(name string) *configschema.Block {
 // either misbehavior on the part of one of the providers or of the provider
 // protocol itself. When returned with errors, the returned schemas object is
 // still valid but may be incomplete.
-func loadSchemas(config *configs.Config, state *states.State, plugins *contextPlugins) (*Schemas, error) {
+func loadSchemas(ctx context.Context, config *configs.Config, state *states.State, plugins *contextPlugins) (*Schemas, error) {
```

Comment:
To be a little more specific about what I meant in the inline reply I just left on another review thread above:

I was imagining this `loadSchemas` function just generating one additional span covering its entire runtime, and nothing more detailed than that for now.

When we get the individual gRPC requests plumbed in (similarly to how we're currently tracing HTTP requests) then those will represent the individual requests, and I personally think any intermediate spans here would be an unnecessary level of detail given that fetching schemas is an implementation detail.